apps/web/pages/api/auth/[...nextauth].tsx

```diff
@@ -166,6 +166,7 @@ export default NextAuth({
           return {
             id: existingUser.id,
             username: existingUser.username,
+            name: existingUser.name,
```

Comment:
This one is important, unlike other code-paths like saml/sso, we weren't passing on name on the credentials one. by passing it, it becomes available to useSession/getSession from next-auth.